### ISSUE TITLE
Create gulp workflow and bring in USWDS assets

### ISSUE THREAD

**Issue by maya** [18F/vote-gov#3]
- [x] Find out which gulp tasks to bring over from: https://github.com/18F/ffd-microsite/blob/staging/gulpfile.js
- [x] Bundle CSS and JS files into `static/assets/` directories to be used by Hugo.
- [x] Install USWDS https://www.npmjs.com/package/uswds

Do we also want to support gulp asset linting of scss and js files?

Should this be two issues?

**Comment by theresaanna** [comment]
I don't see why we shouldn't lint from the start. I can bring over the config from the microsite.

**Comment by rogeruiz** [comment]
Agreed @theresaanna. We also have [the `FFD-Toolkit` project here](https://github.com/18f/ffd-toolkit) that has some starter configuration files. They are more up to date than what is on the microsite at this point.

**Comment by rogeruiz** [comment]
I've brought in all the tasks from the `ffd-microsite` repository and commented out a lot of the code within `gulpfile.js`. Once #6 lands on `master`, let's make another PR with the gulp tasks split into config files.

**Comment by rogeruiz** [comment]
### Proposed split configs for gulp

The following layout is from the web-design-standards repo. I'm including it below as a guideline for the config work on the first checkbox above ☝️. It doesn't necessarily need this many files. 

```
/Users/rogerruiz/Developer/web-design-standards/config/
└── [ 374]  gulp
    ├── [5.7K]  website.js
    ├── [1.4K]  release.js
    ├── [2.1K]  sass.js
    ├── [ 924]  build.js
    ├── [1.6K]  javascript.js
    ├── [ 468]  flags.js
    ├── [2.8K]  doc-util.js
    ├── [ 294]  images.js
    └── [ 297]  fonts.js

1 directory, 9 files
```

As always ❤️, any and all opinions and thoughts are appreciated 💃 👍

**Comment by theresaanna** [comment]
@rogeruiz I didn't see this proposal until now! Are there any places where I deviated from your vision that we should consider?

**Comment by rogeruiz** [comment]
nah, I don't think so. the proposal above was more like a guidance than anything else. what you have in `master` now is a great start and we can iterate on it as we continue. 😀
